app/views/SearchMessagesView/index.tsx

```diff
@@ -272,6 +273,7 @@ class SearchMessagesView extends React.Component<ISearchMessagesViewProps, ISear
 				theme={theme}
 				onPress={() => this.jumpToMessage({ item })}
 				jumpToMessage={() => this.jumpToMessage({ item })}
+				rid={item.rid}
```

Comment:
```suggestion
				rid={message.rid}
```
If the item is a message, this should also be the message shouldn't it? Just to follow the pattern haha

Reply:
haha yeap, will be the same! But you are right 😁